README.md

```diff
@@ -17,7 +17,11 @@ Command-line JSON processing tool
 * Interactive mode 🎉
 * Themes support 🎨
 
-## Install
+## Requirements
```

Comment:
You can download fx as standalone binary. So this requirements section is redundant.

Reply:
Well, I see. Thanks!